docs/content/en/docs-v0.25.x/user-guide/configuration-reference.md

```diff
@@ -316,6 +316,7 @@ One of `yamlField` or `regex` is required.
 | releaseName | string | The release name of helm deployment. By default, the release name is equal to the application name. | No |
 | valueFiles | []string | List of value files should be loaded. | No |
 | setFiles | map[string]string | List of file path for values. | No |
+| apiVersions | []string | Kubernetes api versions used for Capabilities.APIVersions. | No |
```

Comment:
should not update this, please update `/docs-dev` instead, since the next release will be `v0.26.0`

Reply:
Ok, thank you!